Makefile

```diff
@@ -229,6 +229,12 @@ lint: lint-golangci-lint lint-gofumpt lint-govulncheck lint-markdown  ## Run all
 image-image-factory: tailwind  ## Builds image for image-factory.
 	@$(MAKE) registry-$@ IMAGE_NAME="image-factory"
 
+.PHONY: imager-base
+imager-base:
+
+.PHONY: imager-tools
+imager-tools:
```

Comment:
Wondering whether these should really be empty targets here

Reply:
yeh, side effect of kres :sweat_smile: